docs/resource/README.md

```diff
@@ -73,19 +73,20 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 
 ## Service
 
-**Status**: [Stable][DocumentStatus]
-
-**type:** `service`
-
-**Description:** A service instance.
-
 <!-- semconv service -->
 <!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
 <!-- see templates/registry/markdown/snippet.md.j2 -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 
+
+**Status:** ![Stable](https://img.shields.io/badge/-stable-lightgreen)
+
+**type:** `service`
```

Comment:
thanks to your otep I finally understand what it is, I almost tried to remove it - https://github.com/open-telemetry/semantic-conventions/issues/929